etc/kayobe/kolla/globals.yml

```diff
@@ -2,6 +2,11 @@
 docker_yum_baseurl: "{{ stackhpc_repo_docker_url }}"
 docker_yum_gpgkey: "https://download.docker.com/linux/{% raw %}{{ ansible_distribution | lower }}{% endraw %}/gpg"
 
+cloudkitty_tag: wallaby-20220119T122428
+horizon_tag: wallaby-20220119T122428
 manila_tag: wallaby-20211210T140839
 neutron_tag: wallaby-20220104T102739
 neutron_sriov_agent_tag: wallaby-20220110T113538
+prometheus_jiralert_tag: wallaby-20220119T122428
+prometheus_msteams_tag: wallaby-20220119T122428
+prometheus_openstack_exporter_tag: wallaby-20220119T122428
```

Comment:
Note to self: Check this was built. I gave Mark an incomplete list.

Reply:
twas not

Reply:
now built and pushed